sdk/keyvault/Azure.Security.KeyVault.Administration/samples/README.md

```diff
@@ -0,0 +1,15 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-key-vault
+name: Azure.Security.KeyVault.Administration samples for .NET
+description: Samples for the Azure.Security.KeyVault.Administration client library.
+---
+
+# Azure.Security.KeyVault.Administration Samples
+
+- Creating, getting, and deleting role assignments [synchronously](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_RbacHelloWorldSync.md) or [asynchronously](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_RbacHelloWorldAsync.md)
+- [Assigning roles for specific scopes](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample2_RbacScopeAssignment.md) 
```

Comment:
Should we have sync and async samples for this as well? Or not worth it?

Reply:
Since the hello world samples run through the syntax already, I felt it didn't add anything to show the sync examples here also.